build/restore.ps1

```diff
@@ -1,7 +1,14 @@
 $root = $PSScriptRoot
+$packages = "$root\..\packages"
+
 $nuget = & "$root\Get-Nuget.ps1"
 
-& $nuget restore $root\..\PortabilityTools.sln
+if (!(Test-Path $packages))
+{
+    New-Item $packages -ItemType Directory
```

Comment:
I didn't realize -ErrorAction Ignore could be added to any command - very cool to know!